Program.cs

```diff
@@ -54,47 +49,66 @@ public static void RunSample(IAzure azure)
                         .WithNewTopic(topicName, 1024)
                         .Create();
 
-                Utilities.Log("Created service bus " + serviceBusNamespace.Name + " (with queue and topic)");
-                Utilities.Print(serviceBusNamespace);
+                Console.WriteLine("Created service bus " + serviceBusNamespace.Name + " (with queue and topic)");
+
+                PrintServiceBusNamespace(serviceBusNamespace);
 
                 var queue = serviceBusNamespace.Queues.GetByName(queueName);
-                Utilities.Print(queue);
+
+                PrintServiceBusQueue(queue);
 
                 var topic = serviceBusNamespace.Topics.GetByName(topicName);
-                Utilities.Print(topic);
+
+                PrintServiceBusTopic(topic);
 
                 //============================================================
                 // Create 2 subscriptions in topic using different methods.
-                Utilities.Log("Creating a subscription in the topic using update on topic");
+                Console.WriteLine("Creating a subscription in the topic using update on topic");
                 topic = topic.Update().WithNewSubscription(subscription1Name).Apply();
 
                 var subscription1 = topic.Subscriptions.GetByName(subscription1Name);
 
-                Utilities.Log("Creating another subscription in the topic using direct create method for subscription");
+                Console.WriteLine("Creating another subscription in the topic using direct create method for subscription");
                 var subscription2 = topic.Subscriptions.Define(subscription2Name).Create();
 
-                Utilities.Print(subscription1);
-                Utilities.Print(subscription2);
+                PrintSubscriptionsInTopic(subscription1);
+
+                PrintSubscriptionsInTopic(subscription2);
 
                 //=============================================================
                 // Create new authorization rule for queue to send message.
-                Utilities.Log("Create authorization rule for queue ...");
+                Console.WriteLine("Create authorization rule for queue ...");
                 var sendQueueAuthorizationRule = serviceBusNamespace.AuthorizationRules.Define("SendRule").WithSendingEnabled().Create();
-                Utilities.Print(sendQueueAuthorizationRule);
+                StringBuilder builderSendQueueAuthorizationRule = new StringBuilder()

```

Comment:
Please make a separate method for printing rules just like you have for queues and topics

Reply:
@ramya-rao-a I have updated, please review.